### PR TITLE
[v1.x] update armpl version

### PR DESCRIPTION
**Pull request by Zha0q1**
update armpl version to gcc8.2 in an attempt to fix the cd pipeline
tests pipeline here https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/restricted-mxnet-cd%2Fmxnet-cd-pipeline-1.x-zhaoqi/detail/mxnet-cd-pipeline-1.x-zhaoqi/6/pipeline

### CODE REVIEW

**Review by waytrue17 — APPROVED** [review]
LGTM, thanks!

**Review by mseth10 — APPROVED** [review]
LGTM

File: cd/python/docker/Dockerfile [hunk 1]
```diff
@@ -37,9 +37,9 @@ RUN apt-get install -y libgomp1
...
 RUN if [ "$MXNET_VARIANT" = "aarch64_cpu" ] ; then echo "not installing libquadmath0 on aarch64" ; else apt-get install -y libquadmath0 ; fi
 RUN if [ "$MXNET_VARIANT" = "aarch64_cpu" ] ; \
-    then wget https://armkeil.blob.core.windows.net/developer/Files/downloads/hpc/arm-performance-libraries/21-0-0/Ubuntu18.04/arm-performance-libraries_21.0_Ubuntu-18.04_gcc-10.2.tar && \
-    tar -xvf arm-performance-libraries_21.0_Ubuntu-18.04_gcc-10.2.tar && \
-    arm-performance-libraries_21.0_Ubuntu-18.04_gcc-10.2/arm-performance-libraries_21.0_Ubuntu-18.04.sh -a; \
+    then wget https://armkeil.blob.core.windows.net/developer/Files/downloads/hpc/arm-performance-libraries/21-0-0/Ubuntu18.04/arm-performance-libraries_21.0_Ubuntu-18.04_gcc-8.2.tar && \
+    tar -xvf arm-performance-libraries_21.0_Ubuntu-18.04_gcc-8.2.tar && \
+    arm-performance-libraries_21.0_Ubuntu-18.04_gcc-8.2/arm-performance-libraries_21.0_Ubuntu-18.04.sh -a; \
     fi
 ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/arm/armpl_21.0_gcc-10.2/lib
```
Comment (waytrue17) [inline cd/python/docker/Dockerfile:44]: Should we change it to gcc-8.2 as well?

Comment (Zha0q1) [reply cd/python/docker/Dockerfile:44]: right!!

**Pull request merged**